src/components/reservations/ReservationAddModal.tsx

```diff
@@ -37,10 +37,11 @@ const ReservationAddModal = (): JSX.Element | null => {
 
   const closeDialog = () => dispatch(hideReservationAddModal());
 
-  const handleFormCallback = () => {
+  const handleFormCallback = (customerId: string) => {
     setIsLoading(true);
     // TODO: Add operations here
     console.log('form submitted');
+    console.log('customer ID:', customerId);
     console.log('apartment_uuid:', apartment.apartment_uuid);
```

Comment:
I guess these will be removed at some point according to the TODO above, so I'll allow for now.

Reply:
Yep, when we have the backend ready we will remove these. They are here more as notes right now.